lib/runner.js

```diff
@@ -311,9 +311,27 @@ Runner.prototype.run = function() {
       self.on('testFail', restartDriver);
     }
 
+    // We need to save these promises to make sure they're run, but we don't
+    // want to delay starting the next test (because we can't, it's just
+    // an event emitter).
+    var pluginPostTestPromises = [];
+
+    self.on('testPass', function() {
+      pluginPostTestPromises.push(plugins.postTest());
```

Comment:
Might the plugins care if the test passed or failed?  It would be pretty easy to modify `pluginFunFactory` so that if `plugins.postTest(true)` was called then that `true` would be passed along into the plugin's callback.